app/models/customer-statistic.js

```diff
@@ -1,6 +1,6 @@
-import Model, { attr, belongsTo } from "@ember-data/model";
+import Model, { attr } from "@ember-data/model";
 
-export default Model.extend({
-  duration: attr("django-duration"),
-  customer: belongsTo("customer"),
-});
+export default class CustomerStatistics extends Model {
+  @attr("django-duration") duration;
+  @attr("string") name;
```

Comment:
Relationship not needed anymore?

Reply:
NITPICK: Also `@attr("string")` is a type conversion to `String`, so if you know your api already sends a string, this is just unnecessary overhead:
> Transforms
You may find the type of an attribute returned by the server does not match the type you would like to use in your JavaScript code. Ember Data allows you to define simple serialization and deserialization methods for attribute types called transforms. You can specify that you would like a transform to run for an attribute by providing the transform name as the first argument to the attr method. Ember Data supports attribute types of string, number, boolean, and date, which coerce the value to the JavaScript type that matches its name.

From: https://guides.emberjs.com/release/models/defining-models/

Reply:
Not sure if we already talked about this once.

Reply:
The attribute type can safely be assumed as `string` I guess. Just followed the standard of the repo. Thanks for pointing out! TIL!